src/data/wallets/wallet-data.ts

```diff
@@ -0,0 +1,1624 @@
+interface WalletData {
+  last_updated: string
+  name: string
+  image_name: string
+  brand_color: string
+  url: string
+  wallet_live_date: string
+  active_development_team: boolean
+  languages_supported: string[]
+  twitter: string
+  discord: string
+  reddit: string
+  telegram: string
+  ios: boolean
+  android: boolean
+  linux: boolean
+  windows: boolean
+  macOS: boolean
+  firefox: boolean
+  chromium: boolean
+  hardware: boolean
+  open_source: boolean
+  repo_url: string
+  non_custodial: boolean
+  security_audit: string[]
+  scam_protection: boolean
+  hardware_support: boolean
+  walletconnect: boolean
+  rpc_importing: boolean
+  nft_support: boolean
+  connect_to_dapps: boolean
+  staking: boolean
+  swaps: boolean
+  multichain?: boolean
+  layer_2: boolean
+  gas_fee_customization: boolean
+  ens_support: boolean
+  erc_20_support: boolean
+  eip_1559_support: boolean
+  buy_crypto: boolean
+  withdraw_crypto: boolean
+  multisig: boolean
+  social_recovery: boolean
+  onboard_documentation: string
+  documentation: string
+  mpc?: boolean
+}
+
+const walletData: WalletData[] = [
+  {
+    last_updated: "June 22, 2022",
```

Comment:
Why this string format for a date? We should use the same format we do elsewhere (e.g. tutorials) so they can easily be converted to a date object & internationalized.